charts/mattermost-team-edition/README.md

```diff
@@ -19,15 +20,15 @@ cluster using the [Helm](https://helm.sh) package manager.
 ## Prerequisites
 
 - Kubernetes 1.9+ with Beta APIs enabled
-- Helm v2
-- [Tiller](https://rancher.com/docs/rancher/v2.x/en/installation/ha/helm-init/) (the Helm server-side component) installed on the cluster
+- Helm v3
+- [Migrate from Helm v2 to Helm v3](https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/)
 
 ## Installing the Chart
 
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm install --name my-release stable/mattermost-team-edition
+$ helm install my-release stable/mattermost-team-edition
```

Comment:
this is only valid for helm3 not for helm2

maybe you can make a new section for helm3 and how to install using helm3 commands

Reply:
Have used bold font for helm v3 commands. If it is fine, will extend to other charts. Thanks